src/index.js

```diff
@@ -11,7 +11,7 @@ import deleteBlacklist from './helpers/deleteBlacklistTokens';
 
 const app = express(); // setup express application
 
-app.use(logger('dev')); // log requests to the console
+// app.use(logger('dev')); // log requests to the console
```

Comment:
- you have commented the logger in index.js why?

Reply:
Sorry for that, I forgot to uncomment it, I wanted to escape logs when running tests